src/main/java/orca/flukes/ndl/ManifestLoader.java

```diff
@@ -537,19 +468,6 @@ public void ndlInterface(Resource intf, OntModel om, Resource conn,
 				nmInt = "" + RequestSaver.netmaskStringToInt(mask);
 		}
 		
-		// check mirrored storage interfaces
-		if ((ip == null) && (sharedStorageLinkInterfaceEquivalence.containsKey(intf))){
```

Comment:
8. Since sharedStorageLinkInterfaeEquivalence is no longer in use, this if isn't needed either.